src/SelectBox/components/SelectBox.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 
 import {
-    BLUR_DELAY_MS,
```

Comment:
Have you tested this method in every browser? I specifically did what I did to address small issues in several browsers. 

Sent from my iPhone

> On Mar 18, 2016, at 12:06 PM, Jared Lewis <notifications@github.com> wrote:
> 
> In src/SelectBox/components/SelectBox.js:
> 
> >  
> >  import {
> > -    BLUR_DELAY_MS,
> It doesnt use blur to hide anymore, i added a document click
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>